documentation/samplefiles/readme.azureresourceschema.md

```diff
@@ -4,21 +4,19 @@ These settings apply only when `--azureresourceschema` is specified on the comma
 
 ### AzureResourceSchema multi-api
 
-``` yaml $(azureresourceschema) && $(multiapi)
+```yaml
 batch:
-  - tag: schema-yourservicename-YYYY-MM-DD
-
+  - tag: schema-[[ServiceName]]-[[Version]]
 ```
 
 Please also specify `--azureresourceschema-folder=<path to the root directory of your azure-resource-manager-schemas clone>`.
 
-### Tag: schema-yourservicename-YYYY-MM-DD and azureresourceschema
-
-``` yaml $(tag) == 'schema-yourservicename-YYYY-MM-DD' && $(azureresourceschema)
-output-folder: $(azureresourceschema-folder)/schemas
+### Tag: schema-[[ServiceName]]-[[Version]] and azureresourceschema
 
-# all the input files in this apiVersion
-input-file:
-  - Microsoft.YourServiceName/stable/YYYY-MM-DD/YourServiceName.json
+```yaml
+  output-folder: $(azureresourceschema-folder)/schemas
 
+  # all the input files in this apiVersion
+  input-file: 
```

Comment:
no need to put the 2 spaces for indent

Reply:
not only this line... Please check the diff carefully. Don't make unnecessary changes.